packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -96,7 +96,17 @@ export class BlobManager {
         return new BlobHandle(
             `${BlobManager.basePath}/${storageId}`,
             this.routeContext,
-            async () => this.getStorage().readBlob(storageId),
+            async () => {
```

Comment:
You are missing return statement and rethrow, plus comma after error:

```ts
                return this.getStorage().readBlob(storageId).catch((error) => {
                    this.logger.sendErrorEvent(
                        {
                            eventName:"DocumentStorageBlobError",
                            id: storageId,
                        },
                        error,
                    );
                    throw error;
                })
```

Reply:
I'd also rename DocumentStorageBlobError to something more on point, like AttachmentReadBlob (error is imposed from category, and it's about attachment blobs, DocumentStorage is implementation detail so it's less interesting here)